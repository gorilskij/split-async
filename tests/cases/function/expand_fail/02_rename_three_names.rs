#![no_main]

use split_async::split;

#[split(a, b, c)]
async fn func() {}
