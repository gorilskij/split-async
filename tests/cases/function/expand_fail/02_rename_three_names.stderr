error: unexpected token
 --> tests/cases/function/expand_fail/02_rename_three_names.rs:5:15
  |
5 | #[split(a, b, c)]
  |               ^
